community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -394,7 +411,7 @@ def sync_slurm():
         f"reconciling {len(compute_instances)} ({len(all_nodes)-len(compute_instances)}) GCP instances and {len(slurm_nodes)} Slurm nodes ({len(all_nodes)-len(slurm_nodes)})."
     )
     node_statuses = {
-        k: list(v) for k, v in util.groupby_unsorted(all_nodes, find_node_status)
+        k: list(v) for k, v in util.groupby_unsorted(all_nodes, get_node_action)
     }
     if log.isEnabledFor(logging.DEBUG):
```

Comment:
This whole debug-section can be removed.

Reply:
`status` doesn't have property `name`

Reply:
Removing the debug section